### ISSUE TITLE
py.typed is not part of the wheel

### ISSUE THREAD

**Issue by kasium** [PyCQA/pylint#8406]
### Bug description

According to the changelog, 2.17.0 is typed and exposes these. Also the code includes a "py.typed" file. However when extracting the wheel no "py.typed" existss

### Configuration

```ini
n/a
```

### Command used

```shell
n/a
```

### Pylint output

```shell
n/a
```

### Expected behavior

Wheel contains a "py.typed" file

### Pylint version

```shell
pylint 2.17.0
```

### OS / Environment

Linux

**Comment by DanielNoord** [comment]
The changelog does not mention this. `3.0` will expose types and a build from the `main` branch will include this file.

**Comment by Pierre-Sassoulas** [comment]
Should be available in the next alpha version of 3.0.0 that we'll release soon.
